### PR TITLE
feat: polish application ux

### PR DESCRIPTION
**Pull request by logan-r**
### Tickets:

- HCK- #906, #899 

### List of changes:

- Re-arrange application pages, questions and make look match figma design more closely
- Apply very slight gradient background so inputs have contrast
- Small tweaks to pagiation header style (use brown font and 24px line height)
- Fix key bug in pagination header
- Proper spacing of pagination header and get rid of vertical center for applications
- Standardized application page layout and got rid of spacing between buttons
- Get rid of border on school input
- Restyled checkboxes so they have more contrast, subtitle indented and grey instead of italic
- Make dropdowns arrows point downwards
- Allowed for shorter/less-wide versions of inputs (less wide field for number picker, shorter textareas with smaller character limit)
- Fix error message not displaying for previousHackathons question
- Fix display of review page and reduce repetitive code
- Fix page overflow bug that occurred on review page when user entered a long string without spaces for long answer questions
- Fix edit application page form being really squashed
- When application page changes, scroll back to top of page

### Type of change:

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:
image
image
image
image

### CODE REVIEW

File: src/features/Application/ManageApplicationForm.tsx [hunk 1]
```diff
@@ -666,288 +800,149 @@ const ManageApplicationForm: React.FC<IManageApplicationProps> = (props) => {
...
-              {CONSTANTS.DEGREE_LABEL}
-            </H1>
-            <Paragraph fontSize={'16px'} color={theme.colors.purple}>
-              {hackerDetails.application.general.degree}
-            </Paragraph>
+        <div className="container">
+          <H1 marginBottom="16px">Review</H1>
+          <div className="info-text">
+            Your responses are now saved. Please review your application before
+            submitting. You won’t be able to edit your responses later.
```
Comment (jamesxu123) [inline src/features/Application/ManageApplicationForm.tsx:807]: Should we move this to consts?

**Review by jamesxu123 — COMMENTED** [review]
Looks good to me. I think the portfolio page still needs to be adjusted because it looks a little asymmetrical and unbalanced, but I can probably do that in a separate PR.

File: src/features/Application/ManageApplicationForm.tsx (hunk 1 above)
Comment (logan-r) [reply src/features/Application/ManageApplicationForm.tsx]: Yeah, good catch 👍

**Comment by logan-r** [comment]
Yeah I agree, the portfolio page needs to be re-arrange or something. Let's handle that as a separate PR tho.

List of changes since last review:
* Increased spacing between input
* Changed how error messages look and are placed
* Labeled some fields as optional
* Added `0` as an option for number of previous hackathons
* Fix school autoselect dropdown position
* Hide t-shirt if in remote mode
* Changed review description text
* Fix error messages for previous hackathon so their human readable (not complaining about how it's `NaN` when empty, instead just saying required)
* Made it so education background image doesn't jump around when error message appear changing form height

**Review by jamesxu123 — APPROVED** [review]

**Pull request merged**